source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -0,0 +1,156 @@
+---
+title: The Ember Times - Issue No. 129
+author: Chris Ng, Amy Lam, Isaac Lee, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/12/20-the-ember-times-issue-129.html"
+responsive: true
+---
+
+<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+...
+new ember-autofocus-modifier 💡,
+3 new RFCs for template helpers 3️⃣,
+ember-apollo-client v2 released 🎉,
+... 
+
+READMORE
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
```

Comment:
```suggestion
---

## [Octane is Here 🎉](

Ember [3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) was released this week. In addition to the usual new features, improvements and bug fixes, this release also marks the official release of the Octane edition! Read more about what's new and what's changed in Octane in this summary by [Yehuda Katz (@wycats)](https://github.com/wycats).
```

I think @jessica-jordan is already working on this, in which case please totally feel free to discard this as I'm sure Jessica's writing is better than mine 😄 But leaving this here so that in a pinch, we can still at least get a link to the release blog post since that's the major news story in Ember-land this week!

Reply:
Awesome, thank you for the suggestion! I can extend on it a bit if this is useful